packages/libs/src/sdk/api/developer-apps/types.ts

```diff
@@ -6,6 +6,14 @@ import { isApiKeyValid } from '../../utils/apiKey'
 export const CreateDeveloperAppSchema = z.object({
   name: z.string(),
   description: z.optional(z.string().max(128)),
+  imageUrl: z.optional(
+    z
+      .string()
+      .max(2000)
+      .refine((value) => /^(https?):\/\//i.test(value), {
```

Comment:
Feels like this should be more sophisticated to check that it's an image URL, maybe something like this but with whatever extensions we want to support
```
const imageUrlSchema = z.string().regex(
  /^https:\/\/.*\.(jpg|jpeg|png|gif|bmp)$/i,
  "Invalid image URL (supported formats - blah)"
);
```

+ ditto in web Zod validator

Reply:
@schottra I honestly don't think it's worth the misdirection, we tend to keep our zod validations all inline where we use regexes in other places of our codebase

@nicoback2 there's no way to do this because it may not actually have an extension. The best way to check would be request the image, validate its mime type, and even then it's not perfect. Since this is UGC though, it shouldn't matter.

Reply:
Works for me 👍